src/UI/class-recommended-widget.php

```diff
@@ -136,7 +133,15 @@ public function widget( $args, $instance ): void {
 			true
 		);
 
+		wp_register_style(
+			'wp-parsely-recommended-widget',
+			plugin_dir_url( PARSELY_FILE ) . 'build/recommended-widget.css',
+			array(),
```

Comment:
Maybe this makes sense?

```suggestion
			$recommended_widget_script_asset['dependencies'],
```

Reply:
That is very much intentional. If we put the dependencies in, the styles are added too late and don't get rendered correctly.